k8s/trillian/manifest/etcd.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: $NAME-etcd-service
+    labels: &EtcdLabels
+      app.kubernetes.io/name: "$NAME"
+      app.kubernetes.io/component: etcd
+  spec:
+    type: NodePort
+    selector: *EtcdLabels
+    ports:
+      - protocol: TCP
+        port: 2379
+        nodePort: 32379
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: $NAME-etcd-operator
+    labels: *EtcdLabels
+  spec:
+    replicas: 1
+    selector:
+      matchLabels: *EtcdLabels
+    template:
+      metadata:
+        labels: *EtcdLabels
+      spec:
+        serviceAccountName: $ETCD_SERVICE_ACCOUNT
+        containers:
+        - name: trillian-etcd-operator
+          image: $IMAGE_ETCD_OPERATOR
+          command: ["etcd-operator", "-cluster-wide"]
+          env:
+          - name: MY_POD_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+- apiVersion: "etcd.database.coreos.com/v1beta2"
+  kind: "EtcdCluster"
```

Comment:
Should this resource type be listed in the Application resource definition?

Reply:
Done.